tests/network-tests/src/scenarios/full.ts

```diff
@@ -43,23 +43,16 @@ scenario('Full', async ({ job, env }) => {
   // Runtime upgrade should always be first job
   // (except councilJob, which is required for voting and should probably depend on the "source" runtime)
   const councilJob = job('electing council', electCouncil)
-  const runtimeUpgradeProposalJob = env.RUNTIME_UPGRADE_TARGET_WASM_PATH
```

Comment:
Looking at this again, I think this job should be kept as it is designed to test all possible outcomes.
For the runtime upgrade github action specifically, we only need to perform a normal scenario where one proposal is made and passes. So maybe have a flow just for that purpose?